TidyTraitAllomDB.Rmd

```diff
@@ -767,7 +769,7 @@ if (reload_TidyTRY && file.exists(Rdata_RawTRY)){
    n_try_file_list = length(try_file_list)
    file_first      = 1L
    file_last       = n_try_file_list
-   loop_list       = seq(from=file_first,to=n_try_file_list,by=1)
+   loop_list       = seq(from=file_first,to=n_try_file_list)
```

Comment:
Not sure if you dropped the `by=1` part of it or if I added it later, but maybe keep it?